widgetssdk/src/test/java/com/glia/widgets/core/fileupload/FileAttachmentRepositoryTest.kt

```diff
@@ -324,87 +324,87 @@ class LocalAttachmentRepositoryTest {
 
         subjectUnderTest.detachAllFiles()
 
-        assertTrue(subjectUnderTest.localAttachments.isEmpty())
-    }
-
-    @Test
-    fun `addObserver successful when valid argument`() {
-        clearMocks(observer1)
-        subjectUnderTest.clearObservers()
-
-        subjectUnderTest.addObserver(observer1)
-
-        subjectUnderTest.detachAllFiles()
-        verify { observer1.update(any(), any()) }
-    }
-
-    @Test
-    fun `addObserver calls update once when same observer added twice`() {
-        clearMocks(observer1)
-        subjectUnderTest.clearObservers()
-
-        subjectUnderTest.addObserver(observer1)
-        subjectUnderTest.addObserver(observer1)
-
-        subjectUnderTest.detachAllFiles()
-        verify(exactly = 1) { observer1.update(any(), any()) }
-    }
-
-    @Test(expected = NullPointerException::class)
-    fun `addObserver throws when null argument`() {
-        subjectUnderTest.addObserver(null)
-    }
-
-    @Test
-    fun `removeObserver removes when valid argument`() {
-        clearMocks(observer1)
-        subjectUnderTest.clearObservers()
-        subjectUnderTest.addObserver(observer1)
-
-        subjectUnderTest.removeObserver(observer1)
-
-        subjectUnderTest.detachAllFiles()
-        verify(exactly = 0) { observer1.update(any(), any()) }
-    }
-
-    @Test
-    fun `removeObserver does nothing when observer not added`() {
-        subjectUnderTest.clearObservers()
-
-        subjectUnderTest.removeObserver(observer1)
-    }
-
-    @Test
-    fun `removeObserver not throwing when null argument`() {
-        subjectUnderTest.removeObserver(null)
-    }
-
-    @Test
-    fun `clearObservers clears when observers added`() {
-        clearMocks(observer1, observer2)
-        subjectUnderTest.clearObservers()
-        subjectUnderTest.addObserver(observer1)
-        subjectUnderTest.addObserver(observer2)
-
-        subjectUnderTest.clearObservers()
-
-        subjectUnderTest.detachAllFiles()
-        verify(exactly = 0) { observer1.update(any(), any()) }
-        verify(exactly = 0) { observer2.update(any(), any()) }
-    }
-
-    @Test
-    fun `clearObservers does nothing when no observers added`() {
-        subjectUnderTest.clearObservers()
-        subjectUnderTest.clearObservers()
-    }
+        assertTrue(subjectUnderTest.getFileAttachments().isEmpty())
+    }
+
+//    @Test
+//    fun `addObserver successful when valid argument`() {
+//        clearMocks(observer1)
+//        subjectUnderTest.clearObservers()
+//
+//        subjectUnderTest.addObserver(observer1)
+//
+//        subjectUnderTest.detachAllFiles()
+//        verify { observer1.update(any(), any()) }
+//    }
+
+//    @Test
+//    fun `addObserver calls update once when same observer added twice`() {
+//        clearMocks(observer1)
+//        subjectUnderTest.clearObservers()
+//
+//        subjectUnderTest.addObserver(observer1)
+//        subjectUnderTest.addObserver(observer1)
+//
```

Comment:
I see several disabled tests. Is it a leftover?

Reply:
Oh, yes, we don't need those tests.